python/pyspark/testing/utils.py

```diff
@@ -171,3 +209,393 @@ def search_jar(project_relative_path, sbt_jar_name_prefix, mvn_jar_name_prefix):
         raise Exception("Found multiple JARs: %s; please remove all but one" % (", ".join(jars)))
     else:
         return jars[0]
+
+
+# Utilities below are used mainly in pyspark/pandas
+class SQLTestUtils(object):
```

Comment:
Actually there is already `SQLTestUtils` in `pyspark.testing.sqlutils`.

Reply:
Cool, `SQLTestUtils` in `python/pyspark/testing/utils.py` is removed.